src/components/item/item.ts

```diff
@@ -316,12 +319,13 @@ export class Item extends Ion {
     config: Config,
     elementRef: ElementRef,
     renderer: Renderer,
+    plt: Platform,
     @Optional() reorder: ItemReorder
   ) {
     super(config, elementRef, renderer, 'item');
 
     this._setName(elementRef);
-    this._hasReorder = !!reorder;
+    this._reorderSide = reorder ? (isStartSide(reorder.side, plt.isRTL) ? 'start' : 'end') : null;
```

Comment:
we don't need this anymore, also, it is not a good idea to calculate this here, since side can change dynamically?